client/src/components/ImageCard/ImageCard.stories.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import ImageCard from "./ImageCard";
+
+storiesOf("ImageCard", module).add("Default", () => (
+  <ImageCard url="https://wallpaperaccess.com/full/144067.jpg" text={["Hawaii", "May 2020"]} />
```

Comment:
Could we break up the `text` prop into separate props? Maybe something like `bodyText` and `footerText`?